docs/rst/manual/about/release_notes.rst

```diff
@@ -1,6 +1,64 @@
 Release Notes
 =============
 
+Version 2.18
+------------
+
+Highlighted changes
+~~~~~~~~~~~~~~~~~~~
+
+pip installable libres
```

Comment:
Is the highlighted change that `libres` and not `ert` is pip-installable?

Reply:
Yes, ert is not installable in 2.18.